tests/Label/ReadModels/JSON/Repository/EntityTest.php

```diff
@@ -181,7 +139,6 @@ public function it_requires_a_positive_count(): void
             $this->name,
             $this->visibilty,
             $this->privacy,
-            $this->parentUuid,
```

Comment:
Is this test still needed if `count` will be removed?

Reply:
Indeed, but that is done here https://github.com/cultuurnet/udb3-backend/pull/1395